homework/index.js

```diff
@@ -28,9 +30,37 @@
     });
     return elem;
   }
-
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    // eslint-disable-next-line prefer-template
+    const li = createAndAppend('li', ul, { id: 'item' });
+    const repoName = createAndAppend('p', li);
+    repoName.innerHTML = "<span class = 'nameItem'>Repository:</span> ";
+    createAndAppend('a', repoName, {
+      href: repo.html_url,
+      text: repo.name,
+    });
+    const repoDescription = createAndAppend('p', li, { class: 'paragraph' });
+    if (repo.description) {
+      repoDescription.innerHTML =
+        "<span class = 'nameItem'>Description:</span> " + repo.description;
+    } else {
+      repoDescription.innerHTML =
+        "<span class = 'nameItem'>Description: </span> N/A";
+    }
+    const repoForks = createAndAppend('p', li, { class: 'paragraph' });
+    repoForks.innerHTML =
+      "<span class = 'nameItem'>Fork: </span>" + repo.forks_count;
+    const repoDate = createAndAppend('p', li);
+    repoDate.innerHTML =
+      "<span class = 'nameItem'>Updated: </span> " +
+      new Date(repo.updated_at).toLocaleString();
+  }
+
+  const createTitle = document.getElementById('root');
+  const title = createAndAppend('h2', createTitle, { class: 'paragraph' });
+  title.innerText = 'HYF Repositories';
+  function sorting(part1, part2) {
```

Comment:
Use a blank line before a function definition to make them stand out from the rest of the code. This is similar to separating paragraphs in a piece of text with a blank line to improve readability.

Reply:
done :+1:

Reply:
I still see function definitions that are not preceded by a blank line, so it's not done.